src/main/org/deidentifier/arx/aggregates/StatisticsSummary.java

```diff
@@ -533,6 +560,22 @@ private int moveWhileEqual(int index, List<String> values) {
         this.stdDevD = stdDevD;
     }
 
+    
+    /**
+     * Returns the number of distinct values
+     * @return
+     */
+    public int getDistinctNumberOfMeasuresValuesAsInt() {

```

Comment:
There is a copy&paste error in the name of this method. "...NumberOfMeasuresValues....". Please call the method getNumberOfDistinctValuesAsInt().